pkg/security/admission/admission_test.go

```diff
@@ -15,6 +15,7 @@ import (
 	v1kapi "k8s.io/kubernetes/pkg/api/v1"
```

Comment:
@liggitt Shall I backport the tests for `admission` for your PR?

Reply:
it probably won't be an exact copy (since we prefer more restrictive rather than non-mutating), but we do need the tests to start looking at exact matches of PodSecurityContext and SecurityContext, and to check for non-mutation on update